hobbit-gui/gui-serverbackend/src/main/java/de/usu/research/hobbit/gui/rest/ExperimentsResources.java

```diff
@@ -85,18 +100,21 @@
                     // get public experiment
                     Model model = StorageServiceClientSingleton.getInstance().sendConstructQuery(query);
                     if (model != null) {
-                        results.add(RdfModelHelper.createExperimentBean(model, model.getResource(experimentUri)));
+                        if (model.size() > 0) {
```

Comment:
shouldn't it be `if (model != null && model.size() > 0)`? Otherwise the else block is not entered if there is an empty model